test/basic-dataset-chart.html

```diff
@@ -63,9 +61,9 @@
           chartEl.data = data;
           setTimeout(function() {
             expect(chartEl.chart).to.not.be.undefined;
-            expect(chartEl.chart.scale.height).to.be.above(0);
+            expect(chartEl.chart.scales['x-axis-0'].width).to.be.above(0);
             done();
-          }, 10);
```

Comment:
Probably. I just took another look at the docs and found that a `onComplete` callback can be configured that is fired as soon as the chart finished it's animation. Maybe that a better route. http://www.chartjs.org/docs/#chart-configuration-animation-configuration